_data/accessibility-tests/memorable-date.yml

```
@@ -0,0 +1,79 @@
+title: Memorable date
+component_status: pass
+test_items:
+# General tests
+  - summary: Memorable date fields announce their purpose.
+    summary_additional: When you navigate to the select and input fields, you hear that the input expects the day, month, or year.
+    test_status: conditional
```

@amycole501 @alex-hull 

**question:** With this [updated summary](https://github.com/uswds/uswds-site/pull/2919#discussion_r1828017653), should this now be marked as pass?
```suggestion
    test_status: pass
```

---

I think based on the auto populate discussion in that issue, it looked like we landed on marking it conditional still. I think it is truly dependent on the implementation.

---

Test is already marked as conditional in this PR so resolving this convo!